app/views/publish/providers/accredited_providers/edit.html.erb

```diff
@@ -0,0 +1,36 @@
+<% content_for :page_title, title_with_error_prefix("About the accredited provider - #{@accredited_provider.provider_name}", @accredited_provider_form.errors.present?) %>
+
+ <%= form_with(
+        model: @accredited_provider_form,
+        url: publish_provider_recruitment_cycle_accredited_provider_path,
+        method: :put
+      ) do |f| %>
+
+  <%= f.govuk_error_summary %>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+  <% content_for :before_content do %>
+    <%= govuk_back_link_to(publish_provider_recruitment_cycle_accredited_providers_path) %>
+  <% end %>
+
+        <%= f.govuk_text_area(
+          :description,
+          label: { text: t(".title"), size: "l", tag: "h1" },
+          hint: { text: t(".hint") },
+          caption: { text: @accredited_provider.provider_name, size: "l" },
```

Comment:
What happens if you try to update the description with more than 100 words?

Reply:
<img width="1091" alt="image" src="https://github.com/DFE-Digital/publish-teacher-training/assets/50492247/55b444d2-ded7-4d2c-be7e-4a6eec92ff74">